akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -23,6 +23,7 @@ private[stream] object Stages {
 
   object DefaultAttributes {
     val IODispatcher = ActorAttributes.Dispatcher("akka.stream.default-blocking-io-dispatcher")
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name 'IODispatcher' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/vasseurjb/akka/file/2046622393/issues/source?bid=3132306&fileBranchId=3132369#l25)